src/shipit_code_coverage/shipit_code_coverage/taskcluster.py

```diff
@@ -20,14 +21,16 @@ def _get_build_platform_name(platform):
 
 
 def get_last_task(platform):
-    r = requests.get(index_base + 'task/gecko.v2.mozilla-central.latest.firefox.{}'.format(_get_build_platform_name(platform)))
+    index = get_service('index')
+    r = index.findTask(index_base + 'task/gecko.v2.mozilla-central.latest.firefox.{}'.format(_get_build_platform_name(platform)))
```

Comment:
I would expect `index_base` not to be needed anymore.

Reply:
I am not clear on why 'index_base' is not needed, can you help me better understand this ?

Reply:
You can look at the documentation of the taskcluster-client library: https://github.com/taskcluster/taskcluster-client.py#find-indexed-task.